src/model/ofl/fixture.py

```diff
@@ -116,8 +117,10 @@ class UsedFixture:
     """ Fixture in use with a specific mode"""
 
     def __init__(self, name: str, short_name: str, categories: set[Category], comment: str, mode: Mode,
-                 fixture_file: str, mode_index: int, parent_universe: int, name_on_stage: str) -> None:
+                 fixture_file: str, mode_index: int, parent_universe: int, name_on_stage: str = "",
+                 uuid: UUID = uuid4()) -> None:
         """Used Fixture in a specific mode"""
+        self._uuid: UUID = uuid
```

Comment:
Using uuid4() as a default will not work. It will create a default one once on import and use the same one evey time afterwards. Better: `uuid: UUID | None = None` and using `self._uuid: UUID = uuid or uuid4()` later in the setup.